demo/Titanic/Titanic_scenarios/step_defs.resource

```diff
@@ -67,6 +55,12 @@ Titanic sails the Atlantic
     log    Aahh, the fresh air of the open sea!
     Move Titanic out of current area
 
+Titanic crosses Iceberg alley
+    [Documentation]    *model info*
+    ...    :IN: Titanic.location = 'Iceberg alley'
+    ...    :OUT: Titanic.location == 'Atlantic ocean'
+    Check that    Map area where 'Titanic's position' is located    Equals    Atlantic ocean
```

Comment:
Shouldn't something happen here? We change the `Titanic.location` of the model, but we do not interact with the SUT. I know that this is not needed for the scenario to work, because of how it is build up, but wouldn't it make more sense to have an action here? Rather than just a check?

Reply:
This is where you really get into the modelling part. Both options are possible as explained in this [part of the documentation](https://github.com/JFoederer/robotframeworkMBT#how-to-model)
If you choose to do an action here, you also choose to not refine that action further from your specification.